sphinx/util/nodes.py

```diff
@@ -537,15 +535,15 @@ def process_only_nodes(document: Node, tags: "Tags") -> None:
                 node.replace_self(nodes.comment())
 
 
-# monkey-patch Element.copy to copy the rawsource and line
-# for docutils-0.14 or older versions.
-
 def _new_copy(self: Element) -> Element:
+    """monkey-patch Element.copy to copy the rawsource and line
+    for docutils-0.14 or older versions."""
```

Comment:
Unfortunately, this patch is still needed. I found another error on copying system_message node...
refs: https://sourceforge.net/p/docutils/patches/165/